Doc/library/pathlib.rst

```diff
@@ -886,9 +886,11 @@ conforming to :rfc:`8089`.
       >>> p.as_uri()
       'file:///c:/Windows'
 
-   For historical reasons, this method is also available from
-   :class:`PurePath` objects. However, its use of :func:`os.fsencode` makes
-   it strictly impure.
+   .. deprecated-removed:: 3.14 3.19
```

Comment:
No opposition, but why 3.19? There's nothing currently scheduled for removal that far out (https://docs.python.org/dev/whatsnew/3.14.html#deprecated), and I can't imagine `PurePath.as_uri()` is very common.

Reply:
https://peps.python.org/pep-0387 was recently updated to prefer 5-year deprecation periods IIUC, and I don't think keeping it around blocks further progress on anything.